UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.Common/Models/Bess/BessConfig.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Newtonsoft.Json;
+
+namespace UKHO.PeriodicOutputService.Common.Models.Bess
+{
+    [JsonObject(ItemNullValueHandling = NullValueHandling.Ignore)]
+    public class BessConfig
```

Comment:
As other comment should class be capitalised as 'BESSConfig'

Reply:
keep it consistent with existing code.